### PR TITLE
#168781663 set sequelize and postgres sql

### PR DESCRIPTION
**Pull request by ngireric123**
### What does this PR do?
* Setup Sequelize and PostgresSQL
### Description of Task to be completed?
* install sequelize-cli
* install --save path
* install --save sequelize pg pg-hstore
* sequelize initialization
* install --save dotenv
* creating config.js
* setting dotenv file
* creating Database, modal, and Migration
* Create Services, Controllers, Routes, and Utilities
* edit controller, services and model file
*add seeders
### How should this be manually tested?
* clone repository
* Run npm install
* create .env file
* run sequelize db:migrate
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
#168781663
### Screenshots (if appropriate)
N/A
### Questions:
N/A

### CODE REVIEW

**Review by Cheza-Dzabala — COMMENTED** [review]
@ngireric123 Please correct ` hound ` warnings

**Review by Cheza-Dzabala — COMMENTED** [review]
Screen Shot 2019-10-02 at 16 56 27

Please resolve this first

**Review by alainmateso — CHANGES_REQUESTED** [review]
Please make use of .env.example and provide all necessary environment variables for others to be able to run this.

File: src/database/models/index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+const fs = require('fs');
```
Comment (sabin18) [inline src/database/models/index.js]: you did a great job but it is better to use ES6 by using import here .

**Review by sabin18 — CHANGES_REQUESTED** [review]

**Review by sabin18 — APPROVED** [review]
it looks good.

**Review by Cheza-Dzabala — DISMISSED** [review]
All good

**Review by alainmateso — DISMISSED** [review]

**Review by Cheza-Dzabala — APPROVED** [review]

**Review by alainmateso — APPROVED** [review]

**Pull request merged**